.cmake/Modules/PackageUtils.cmake

```diff
@@ -79,7 +79,7 @@ macro (cr_find_package _PKG)
 
     if (${_PKG_UP}_PKG_FOUND)
       if (${_PKG_UP}_PKG_LIBRARY_DIRS)
-        link_directories(${PCRE_PKG_LIBRARY_DIRS})
```

Comment:
This seems to be an error in `PackageUtils.cmake`. Replace `link_directories(${PCRE_PKG_LIBRARY_DIRS})` with `link_directories(${${_PKG_UP}_PKG_LIBRARY_DIRS})`.